notebooks/official/migration/UJ14 Vertex SDK AutoML Video Classification.ipynb

```diff
@@ -8,7 +8,7 @@
       },
```

Comment:
<p>Do not change the copyright year</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/840/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>